### DISCUSSION TITLE
Has webots-R2023 been fully tested and supported for CentOS systems?

### DISCUSSION THREAD

**Discussion by mhwang2000** [cyberbotics/webots#6133] [General]
It appears webots-R2023 has been  tested only for Ubuntu 20.04 & 22.04. Has it been fully tested and supported for CentOS systems? Thanks.

**Comment by omichel** [comment]
No, we do not test other Linux distributions than Ubuntu 20.04 & 22.04.
However, it is very likely to work well on many other Linux distributions out-of-the-box.
In the worst case, you can always [recompile Webots from the source](https://github.com/cyberbotics/webots/wiki/Linux-installation/).
